pkg/otlp/logs/go.mod

```diff
@@ -0,0 +1,35 @@
+module github.com/DataDog/opentelemetry-mapping-go/pkg/otlp/logs
+
+go 1.19
+
+require (
+	github.com/DataDog/datadog-api-client-go/v2 v2.13.0
+	github.com/DataDog/opentelemetry-mapping-go/pkg/otlp/attributes v0.2.3
```

Comment:
You need to add a replace statement for this, similar to https://github.com/DataDog/opentelemetry-mapping-go/blob/efedb643325cbc0ab38912cb29bab350c320cbbf/pkg/otlp/metrics/go.mod#L44

This will ensure that we are testing with the latest version of the code instead of the latest released version